src/examples/liquid_fill_gauge/liquid_fill_gauge.ts

```diff
@@ -250,7 +250,20 @@ const vis: LiquidFillGaugeVisualization = {
     this.svg.attr('height', element.clientHeight - 20)
 
     // @ts-ignore
-    d3.liquidfillgauge(this.svg, value, gaugeConfig)
+    if (details["print"]) {
```

Comment:
This looks good to, but can be written a little more compactly if you like (using some new syntax in ES6).

```
if (details['print']) {
  Object.assign(guageConfig, {
    valueCountUp: false,
    waveAnimationTime: 0,
    // and the rest
  })
}
d3.liquidfillgauge(this.svg, value, gaugeConfig)
```